crates/gradbench/src/stats.rs

```diff
@@ -198,11 +198,89 @@ impl<R: BufRead, F: CreateFile> Scorer<R, F> for ScorerClassic {
     }
 }
 
+/// A scorer for evals that contain multiple semantically equivalent
+/// but operationally different functions, like particle and saddle.
+/// We assume only one workload per function.
+///
+/// The point of these evals is to investigate the consequences of
+/// different implementation choices. In a real setting, one would
+/// presumably pick the best one, so the score is reported as the
+/// minimum runtime achieved.
+#[derive(Serialize)]
+struct ScorerParticleSaddle {
+    /// The average duration for each tool (outer keys) and function (inner keys).
+    tools: BTreeMap<String, IndexMap<Rc<str>, Duration>>,
+}
```

Comment:
Thanks for the nice documentation! Since (like `ScorerClassic`) this scorer is based around an idea rather than only being suitable for `particle` and `saddle`, can you change the name to be more general, to be consistent with how `ScorerClassic` is already named?